### PR TITLE
Copy https://github.com/openvinotoolkit/openvino/blob/cabb40638a386d32e704d1001315ded76598d5ff/src/bindings/python/src/openvino/__init__.py

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: .github/workflows/test_precommit.yml [hunk 1]
```diff
@@ -45,11 +45,11 @@ jobs:
     - name: Check style with black
       run: |
         source venv/bin/activate
-        black --check .
+        black --exclude model_api/python/openvino/__init__.py --check .
```
Comment (sovrasov) [inline .github/workflows/test_precommit.yml:48]: Can we add skip directives to the file? Passing ignore list to command line is not very convenient and scalable

Comment (Wovchena) [reply .github/workflows/test_precommit.yml:48]: black reads `pyproject.toml`. But having `pyproject.toml` would suggest that we support building a package from it

**Review by sovrasov — APPROVED** [review]

**Comment by AlexKoff88** [comment]
@ilya-lavrenov, fyi. It seems like the namespace remains the same.

@Wovchena, when are you planning to change the namespace and remove `openvino.` from the beginning?

**Comment by Wovchena** [comment]
Everyone I discussed this topic with prefers to keep `openvino.`, so I'm going to keep it.

**Review by AlexKoff88 — APPROVED** [review]

**Pull request merged**